src/main/java/com/spotify/docker/client/DockerClient.java

```diff
@@ -286,9 +286,13 @@ InputStream save(String image, AuthConfig authConfig)
   /**
    * Pull a private docker container image.
    *
-   * @param image The image to pull.
+   * @param image      The image to pull.
    * @param authConfig The authentication config needed to pull the image.
+<<<<<<< HEAD
```

Comment:
Hi,
Developer is on holiday until january 4.
We will decide the code next year:-)

Thanks

Il mercoledì 23 dicembre 2015, David Xia <notifications@github.com> ha
scritto:

> In src/main/java/com/spotify/docker/client/DockerClient.java
> <https://github.com/spotify/docker-client/pull/307#discussion_r48367397>:
>
> >     * @param authConfig The authentication config needed to pull the image.
> > +<<<<<<< HEAD
>
> woops :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/docker-client/pull/307/files#r48367397>.
>


-- 
Damiano Pezzotti
...................................... . .
Software Architect | Alambitco
mobile +39 333 2116258
fax +39 02 700432209
email damy@alambitco.com
web www.alambitco.com
flickr www.flickr.com/photos/damianopezzotti
linkedin www.linkedin.com/in/damianopezzotti
pgp id 0x12DFD989